DOCUMENTATION.md

```diff
@@ -519,6 +520,8 @@ You can only get the status of messages that are 7 days old or newer.
 |#Pending virus check|GOV.UK Notify has not completed a virus scan of the precompiled letter file.|
 |#Virus scan failed|GOV.UK Notify found a potential virus in the precompiled letter file.|
 |#Validation failed|Content in the precompiled letter file is outside the printable area. See the [GOV.UK Notify letter specification](https://www.notifications.service.gov.uk/using-notify/guidance/letter-specification) for more information.|
+|#technical-failure|GOV.UK Notify had an unexpected error while sending the letter to our printing provider.|
```

Comment:
At the moment, this table only contains the statuses that are specific to precompiled letters, so `accepted` and `received` are not included. If we were to stick to that pattern, then we only need `technical-failure` and `permanent-failure` in the table above. Though `permanent-failure` would be very unlikely (impossible?) with templated letters, so maybe that one belongs here 🤔

Reply:
Oh dear... I've updated all the other doc the same way as this one.